app/views/posts/new.html.erb

```diff
@@ -1,6 +1,6 @@
 <h2>Create new post</h2>
 
-<%= form_with model: [@user, @post] do |form| %>
+<%= form_with model: [@user, @post], url: user_posts_path(@user) do |form| %>
```

Comment:
#### Optional

- [ ] 👋 Hi team! I stumbled upon a bug while attempting to create a post. Creating a Post is a crucial feature for your app, so I'd strongly recommend looking into it. 🚀

![image](https://github.com/ahmidou09/blog_app/assets/98453979/4f95af0e-a349-4b9d-8134-afb64b9b56cd)




- [ ] It seems that the issue might be related to the way you're handling the creation of the `Post object` in your create action. You're creating the` @post object` twice, and the second time you're overwriting the changes made to `@post.comments_counter `and `@post.likes_counter.`


#### Suggestion

- [ ] You can use this approach to address the problem

```
def create
  @post = current_user.posts.new(post_params)
  @post.comments_counter = 0
  @post.likes_counter = 0

  if @post.save
    current_user.update(post_counter: current_user.posts.count)
    redirect_to user_posts_path(current_user), notice: 'Post created successfully.'
  else
    render :new, status: :unprocessable_entity
  end
end


```

#### Keep up the good work 👏

Reply:
Thank you for taking the time to review, @frank1738